### ISSUE TITLE
Foreign key constraint violated

### ISSUE THREAD

**Issue by 0-Il** [bepaald/signalbackup-tools#118]
My issue is a variation of https://github.com/bepaald/signalbackup-tools/issues/107

My input and output: 
```
signalbackup-tools_win.exe [input] [passphrase] --runsqlquery "pragma foreign_key_check"

done!

WARNING Foreign key constraint violated.
--------------------------
| table | parent  | fkid |
--------------------------
| call  | message | 2    |
--------------------------
 * Executing query: pragma foreign_key_check
table|rowid|parent|fkid
call|1|message|2
call|2|message|2
```

What should I do further?

**Comment by dreamflasher** [comment]
@0-Il I am running into the same issue with Molly and was looking for the issue you created over at Molly, but couldn't find it. Where did you post this?

**Comment by 0-Il** [comment]
@dreamflasher, I didn't create any issue at Molly, I created in Signal instead: https://github.com/signalapp/Signal-Android/issues/12948

**Comment by dreamflasher** [comment]
Ah, thanks @0-Il ! Fair point, it's unclear if it's a Signal or Molly issue, while they claim they fixed it, I nevertheless ran into the same issue today, with a backup from Molly and latest Molly version. As I never had any issue like this with Signal, I am switiching back to Signal, unfortunately.

**Comment by 0-Il** [comment]
@dreamflasher

> I am switiching back to Signal

Fair enough, too. I had been using Molly until encountering this issue of mine. After I fixed everything, I moved to Signal too, to eliminate any possibility of incompatibilities or bugs that may arise, by using an official app. However, Molly's developer told me that Molly and Signal are 100% compatible, and I will take their word for it. 

I used Signal successfully until I wiped Google Play Services and everything else related to Google from my phone via 'universal-android-debloater'. I won't go into details here, but afterwards, Signal could no longer function normally without Play Services (and there's no fix for that) and I had to move to FOSS version of Molly (Molly-FOSS). If not for Molly (or TwinHelix's fork of Signal, but Molly updates much more often and overall is leaps and bounds better), I would be locked out of Signal, so huge thanks to Molly for existing.
